testing/validation.ts

```diff
@@ -184,6 +184,8 @@ function checkPropertyTypeAndCodaType<ResultT extends any>(
         case undefined:
           // no need to coerce current result type
           return [];
+        case ValueHintType.CodaInternalRichText:
+          return [{message: `CodaInternalRichText is not supported in external packs.`}];
```

Comment:
Interesting, so are we able to hard-block here in the SDK because the cross-doc getSchema call never calls our validation code? Not sure if that's great if so haha but love that we're able to reliably block other usage.

Reply:
Yeah, that's the idea.  Seems to work properly in my prototype.

Reply:
Though to be clear, this is only when running client-side pack validation of runtime results.  It doesn't block uploading such a pack.  I can bifurcate some of our validation logic to differentiate internal vs. external usage with more work if that's desired.

Reply:
Oh, can you make sure this gets validated in upload_validation.ts? That gets called both on the client and on the upload.

Reply:
Yeah, I'll work on a `superRefine` to catch this case if I can wrestle it to the ground.